client/src/components/EmailAlertSignup.tsx

```diff
@@ -64,20 +73,52 @@ const BuildingSubscribeWithoutI18n = (props: BuildingSubscribeProps) => {
   return (
     <I18n>
       {({ i18n }) => (
-        <div className="table-content building-subscribe">
-          {!(subscriptions && !!subscriptions?.find((s) => s.bbl === bbl)) ? (
-            <button
-              className="button is-primary"
-              onClick={() => subscribe(bbl, housenumber, streetname, zip, boro)}
-            >
-              <Trans>Get updates</Trans>
-            </button>
-          ) : verified ? (
-            showSubscribed()
-          ) : (
-            showEmailVerification(i18n)
-          )}
-        </div>
+        <>
+          <div className="table-content building-subscribe">
+            {!(subscriptions && !!subscriptions?.find((s) => s.bbl === bbl)) ? (
+              <button
+                className="button is-primary"
+                onClick={() =>
+                  subscriptions.length <= SUBCSCRIPTION_LIMIT
+                    ? subscribe(bbl, housenumber, streetname, zip, boro)
+                    : setShowSubscriptionLimitModal(true)
+                }
+              >
+                <Trans>Get updates</Trans>
+              </button>
+            ) : verified ? (
+              showSubscribed()
+            ) : (
+              showEmailVerification(i18n)
+            )}
+          </div>
+
+          <Modal
+            key={1}
+            showModal={showSubscriptionLimitModal}
+            width={40}
+            onClose={() => setShowSubscriptionLimitModal(false)}
+          >
+            <Trans render="h4">You have reached the maximum number of building updates</Trans>
+            <Trans>
+              At this time we can only support {SUBCSCRIPTION_LIMIT} buildings in each email. Please
```

Comment:
in the deploy preview, looks like `SUBSCRIPTION_LIMIT` is just showing up as the variable name. I've seen this happen with variables like `email` but this one is a constant value. does this happen on your end? 
<img width="575" alt="Screenshot 2024-02-26 at 11 16 13 PM" src="https://github.com/JustFixNYC/who-owns-what/assets/17574626/844628e5-d432-493d-8804-d759e06db6fc">

Reply:
it works for me locally - i think it's just the same thing of needing to do lingui extract, but I thought I'd wait to do that all at once before merging into master